<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>switchEditForm</class>
 <widget class="QWidget" name="switchEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>674</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background: rgb(0, 151, 157);
padding: 3px
</string>
     </property>
     <property name="text">
      <string>Edit switch</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>State</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>Arduino pin:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="SW_PIN_comboBox">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Switch ON actions:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_15">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Switch OFF actions</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="4" column="1">
      <widget class="QTableWidget" name="SWON_CMDS_TABLE">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>75</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>Command</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QTableWidget" name="SWON_DREFS_TABLE">
       <column>
        <property name="text">
         <string>Dataref</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Set to Value</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Send XPlane Commands: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="SWON_ADDCMD_BTN">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/newPrefix/plusIcon.png</normalon>
          </iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="SWON_RMCMD_BTN">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/newPrefix/minusIcon.png</normaloff>:/newPrefix/minusIcon.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Set Dataref: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="SWON_ADDDREF_BTN">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/newPrefix/plusIcon.png</normaloff>:/newPrefix/plusIcon.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="SWON_RMDREF_BTN">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/newPrefix/minusIcon.png</normaloff>:/newPrefix/minusIcon.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="10" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="6" column="0">
      <widget class="QTableWidget" name="SWOFF_CMDS_TABLE">
       <column>
        <property name="text">
         <string>Command</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QTableWidget" name="SWOFF_DREFS_TABLE">
       <column>
        <property name="text">
         <string>Dataref</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Set to Value</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Send XPlane Command: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="SWOFF_ADDCMD_BTN">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/newPrefix/plusIcon.png</normalon>
          </iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="SWOFF_RMCMD_BTN">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/newPrefix/minusIcon.png</normalon>
          </iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Set Dataref:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="SWOFF_ADDDREF_BTN">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/newPrefix/plusIcon.png</normalon>
          </iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="SWOFF_RMDREF_BTN">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/newPrefix/minusIcon.png</normalon>
          </iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="SWEDIT_nameLineEdit">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="SWEDIT_IDlineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>SWON_ADDCMD_BTN</sender>
   <signal>clicked()</signal>
   <receiver>switchEditForm</receiver>
   <slot>addSwitchOnCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SWON_RMCMD_BTN</sender>
   <signal>clicked()</signal>
   <receiver>switchEditForm</receiver>
   <slot>rmSwitchOnCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SWON_ADDDREF_BTN</sender>
   <signal>clicked()</signal>
   <receiver>switchEditForm</receiver>
   <slot>addSwitchOnDataref()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SWON_RMDREF_BTN</sender>
   <signal>clicked()</signal>
   <receiver>switchEditForm</receiver>
   <slot>rmSwitchOnDataref()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SWOFF_ADDCMD_BTN</sender>
   <signal>clicked()</signal>
   <receiver>switchEditForm</receiver>
   <slot>addSwitchOffCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SWOFF_RMCMD_BTN</sender>
   <signal>clicked()</signal>
   <receiver>switchEditForm</receiver>
   <slot>rmSwitchOffCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SWOFF_ADDDREF_BTN</sender>
   <signal>clicked()</signal>
   <receiver>switchEditForm</receiver>
   <slot>addSwitchOffDataref()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SWOFF_RMDREF_BTN</sender>
   <signal>clicked()</signal>
   <receiver>switchEditForm</receiver>
   <slot>rmSwitchOffDataref()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SWON_CMDS_TABLE</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>switchEditForm</receiver>
   <slot>editXPCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SWOFF_CMDS_TABLE</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>switchEditForm</receiver>
   <slot>editXPCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SWEDIT_nameLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>switchEditForm</receiver>
   <slot>updateSwitchXMLdata()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SW_PIN_comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>switchEditForm</receiver>
   <slot>updateSwitchXMLdata()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SWON_CMDS_TABLE</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>switchEditForm</receiver>
   <slot>updateSwitchXMLdata()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>658</x>
     <y>-10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SWOFF_CMDS_TABLE</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>switchEditForm</receiver>
   <slot>updateSwitchXMLdata()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addSwitchOnCommand()</slot>
  <slot>rmSwitchOnCommand()</slot>
  <slot>addSwitchOnDataref()</slot>
  <slot>rmSwitchOnDataref()</slot>
  <slot>addSwitchOffCommand()</slot>
  <slot>rmSwitchOffCommand()</slot>
  <slot>addSwitchOffDataref()</slot>
  <slot>rmSwitchOffDataref()</slot>
  <slot>editXPCommand()</slot>
  <slot>updateSwitchXMLdata()</slot>
 </slots>
</ui>
